docs/body.md

```diff
@@ -459,6 +459,16 @@ This ZenPack still supports the previous configuration, where messages were sent
 
 For new openstack installs, we recommend using the RabbitMQ-Ceilometer endpoint, which is what will be reported by `openstack_amqp_config`.
 
+When the RabbitMQ-Ceilometer service is added to Control Center, two IP Assignments will be added as well.  Upon creation, they will have no IP address, only a port number.  You must add the IP address by following these steps:
```

Comment:
I'm actually not sure that it's supposed to be necessary to do this step manually- it might be a CC glitch of some sort, in which case we might want to phrase it more they should check this, and add the assignment if it's missing.   

I'm not certain though- maybe stevepc could clarify.